README.md

```diff
@@ -14,9 +14,8 @@ ___
 * RefApp 2.x Workflow Tests (Chrome) [BDD Chrome](https://ci.openmrs.org/browse/CONTRIB-QA) ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/CONTRIB-QA)
   * [![RefApp 2.x Clinical Visit Test](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-clinical-visit.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-clinical-visit.yml)
   * [![RefApp 2.x Vitals And Triaging Test](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-vitals-and-triaging.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-vitals-and-triaging.yml) 
+  * [![RefApp 2.x Encounter Management Test](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-encounter.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-encounter.yml)
   * Tests in development:
-    * Managing User Accounts
```

Comment:
Please remove things where applicable , this was supposed to be removed in   https://github.com/openmrs/openmrs-contrib-qaframework/pull/78

Reply:
Did you get a chance to see my comment [here](https://github.com/openmrs/openmrs-contrib-qaframework/pull/78#discussion_r692186811) ?

Reply:
Please make things look standard, that commit is supposed to be removed from  RATEST-157

Reply:
See my comment [here](https://github.com/openmrs/openmrs-contrib-qaframework/pull/78#discussion_r692201634)